### PR TITLE
Fixes #6566: promises.cf's \"bundle agent check_cron_daemon\" is not systemd aware

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/6566

### CODE REVIEW

**Comment by amousset** [comment]
Do not merge for now, this will be improved.

**Comment by amousset** [comment]
PR updated.

File: initial-promises/node-server/promises.cf [hunk 1]
```diff
@@ -365,30 +365,44 @@ bundle agent check_uuid
 }
 
 #######################################################
+# Match all variants of cron proccesses
+
+body process_select cron_bin
+{
+  command => "(/usr/sbin/)?crond?( -n| -f)?";
```
Comment (amousset) [inline initial-promises/node-server/promises.cf]: This allows matching the complete COMMAND, and not only the end, allowing not to match anacron for example.

**Comment by Kegeruneku** [comment]
<stamp> !

**Pull request merged**